enhancements/insights/rapid-recommendations.md

```diff
@@ -0,0 +1,552 @@
+  These use cases should be addressed separately.
+
+ 
+## Proposal
+
+The proposal is based on a concept already used by the 
+[Conditional data gathering](../insights/conditional-data-gathering.md) feature of the Insights Operator:
+Before the Insights Operator starts creating a new archive (i.e. every 2 hours), 
+it will download remote configuration from a service on console.redhat.com 
+and gather data specified in the configuration. 
+The Observability Intelligence team (former CCX) will maintain the remote configuration in a repository 
+and deploy it to console.redhat.com whenever it changes.
+
+### Workflow Description
+
+This proposal introduces a new workflow for requesting new data to be collected by the Insights Operator.
+
+A **data requester** is a person who analyzes or develops tools to analyze Insights Operator archives
+uploaded to Red Hat.
+
+1. A data requester identifies the need for new data in the Insights Operator archive.
+2. The data requester describes the data using a format required by the remote configuration service.
+3. The data requester creates a pull request to the repository where the remote configuration is maintained.
+4. The pull request is reviewed (and either approved or denied) 
+   by a member of the Observability Intelligence (former CCX) team.
+5. When the pull request is merged, the new remote configuration is deployed 
+   to the remote configuration service on console.redhat.com.
+6. Insights Operators in connected clusters download the new remote configuration 
+   and start collecting the new data.
+7. The data requester makes use of the new data in newly incoming Insights Operator archives.
+
+### API Extensions
+
+This proposal does not add any API extension.
+
+### Topology Considerations
+
+#### Hypershift / Hosted Control Planes
+
+This proposal does not require any specific details to account for the HyperShift. 
+For now, we can simply extend the remote configuration file to 
+request gathering of specific HyperShift resources.
+
+#### Standalone Clusters
+
+This proposal is relevant to standalone (and especially connected) clusters. 
+Standalone cluster retrieves the remote configuration (based on its OCP version) and collects data accordingly. 
+
+#### Single-node Deployments or MicroShift
+
+This proposal may lead to increased resource consumption by the Insights Operator 
+(described in the [Excessive use of cluster resources](#excessive-use-of-cluster-resources) section), 
+but the operator can be disabled in these environments or in case of problems. 
+
+### Implementation Details/Notes/Constraints 
+
+#### Remote Configuration Service
+
+The Insights Operator configuration will replace the current [conditional gathering endpoint](../insights/conditional-data-gathering.md)
+
+```bash
+https://console.redhat.com/api/gathering/gathering_rules.json
+```
+
+with a new one:
+
+```bash
+https://console.redhat.com/api/gathering/v2/%s/gathering_rules.json
```

Comment:
The Monitoring team is considering a proposal for a very similar mechanism for telemetry metrics. Is it feasible to namespace the endpoint further? E.g. `https://console.redhat.com/api/remote_config/gathering/v2/%s/gathering_rules.json`
A telemetry endpoint could then use `https://console.redhat.com/api/remote_config/telemetry` or similar.